oshi-core/src/main/java/oshi/hardware/common/AbstractNetworkIF.java

```diff
@@ -75,10 +75,9 @@
      * Construct a {@link NetworkIF} object backed by the specified
      * {@link NetworkInterface}.
      *
-     * @param netint
-     *            The core java {@link NetworkInterface} backing this object.
-     * @throws InstantiationException
-     *             If a socket exception prevents access to the backing interface.
+     * @param netint The core java {@link NetworkInterface} backing this object.
+     * @throws InstantiationException If a socket exception prevents access to the
+     *                                backing interface.
```

Comment:
Try to avoid whitespace changes like this.  See the oshi-dist src/main/resources folder for an XML formatter file for Eclipse (it can be used with intelliJ also).